scripts/run_dev.sh

```diff
@@ -40,7 +40,7 @@ if [ "$fill_method" = "indexer" ]; then
     }' | jq '.result')
 
     # Calculate the starting block
-    starting_block=$((latest_block - 100))
+    starting_block=$((latest_block - 1000))
```

Comment:
let's make that configurable in the script with a default, if you can also add that it takes the apibara key from env var by default